File: CS/DragDropExample/GridCellData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DragDropExample
{
    public class GridCellData
    {
        string content;
        public GridCellData(string text)
        {
            content = text;
        }
        public string Content
        {
            get { return content; }
            set { content = value; }
        }
    }
}

File: VB/DragDropExample/GridCellData.vb
Namespace DragDropExample

    Public Class GridCellData

        Private contentField As String

        Public Sub New(ByVal text As String)
            contentField = text
        End Sub

        Public Property Content As String
            Get
                Return contentField
            End Get

            Set(ByVal value As String)
                contentField = value
            End Set
        End Property
    End Class
End Namespace
